### DISCUSSION TITLE
Database Access fails: Access to read lock denied

### DISCUSSION THREAD

**Discussion by bertrand-leroy** [gravitational/teleport#8079] [Q&A]
Hello

i'm trying to setup database access to a self-hosted mysql server. The database is listed by `tsh db ls` but connection to the server fails.

Going through the logs, I see these two lines coming every couple of seconds:
_On the teleport proxy server:_
`INFO [RBAC]      Access to read lock in namespace default denied to roles Db,default-implicit-role: no allow rule matched. services/role.go:2148`

_On the database server:_
`WARN [DB:SERVIC] Restart watch on error: watcher is closed. resource-kind:lock services/watcher.go:187`

I can't find any reason for these errors. Does anybody have an idea?
Thanks a lot!

PS: running Teleport v7.1.0

**Comment by webvictim** [comment]
Are all components in your Teleport setup running v7.1.0 here?
